debian/control

```diff
@@ -134,44 +339,19 @@ Recommends:
     gnome-power-manager,
     gnome-weather,
     gucharmap,
-    ibus-table-emoji,
-    libavcodec58,
-    libreoffice-calc,
-    libreoffice-impress,
-    libreoffice-writer,
-    pop-shop,
-    pop-transition,
-    popsicle-gtk,
     seahorse,
     simple-scan,
     totem,
     yelp,
-# Last to replace mime information
-    file-roller,
-    com.github.donadigo.eddy,
-# Programs
-    popsicle,
 # Plugins
-    brltty,
-    fonts-noto-color-emoji,
     gnome-remote-desktop,
     gnome-shell-extension-prefs,
     gstreamer1.0-vaapi,
     libreoffice-gnome,
-    libreoffice-ogltrans,
-    network-manager-config-connectivity-pop,
-    sessioninstaller,
-# Desktop
-    dbus-broker,
-    hidpi-daemon,
-    flatpak,
-    system76-scheduler,
-    touchegg,
-Conflicts: system76-desktop, pulseaudio, pipewire-media-session
-Description: Pop Desktop Metapackage
 
-Package: pop-desktop-raspi
+Package: pou-desktop-raspi
```

Comment:
Changing `pop-desktop` to `pou-desktop` looks like a typo.

Reply:
Sure is. Fixed. Thank you